src/transformers/models/gpt_bigcode/modeling_gpt_bigcode.py

```diff
@@ -129,23 +124,24 @@ def __init__(self, config, is_cross_attention=False, layer_idx=None):
             if self.is_mqa:
                 raise NotImplementedError(f"attention_type {self.attention_type}  for cross_attention")
 
-            self.c_attn = torch.nn.Linear(self.embed_dim, 2 * self.embed_dim)
-            self.q_attn = torch.nn.Linear(self.embed_dim, self.embed_dim)
+            self.c_attn = Linear(self.embed_dim, 2 * self.embed_dim)
```

Comment:
@younesbelkada Not sure about this formatting, that's what I would use but other models seem to use `nn.Linear`. This prevents several methods/classes below from just being a copy of GPT2

Reply:
Yes let's use `nn.Linear` to be consistent